ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/CommandlineParser.java

```diff
@@ -14,34 +14,35 @@
 
 import tech.pegasys.ethsigner.core.InitializationException;
 
-import java.io.PrintStream;
+import java.io.PrintWriter;
 import java.util.List;
 
 import com.google.common.collect.Lists;
 import org.apache.logging.log4j.Level;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import picocli.CommandLine;
-import picocli.CommandLine.ExecutionException;
 import picocli.CommandLine.Help.Ansi;
 import picocli.CommandLine.ParameterException;
-import picocli.CommandLine.RunLast;
 
 public class CommandlineParser {
 
   private static final Logger LOG = LogManager.getLogger();
 
   private final List<SignerSubCommand> signers = Lists.newArrayList();
   private final EthSignerBaseCommand baseCommand;
-  private final PrintStream output;
+  private final PrintWriter stdOut;
+  private final PrintWriter stdErr;
 
   public static final String MISSING_SUBCOMMAND_ERROR = "Signer subcommand must be defined.";
   public static final String SIGNER_CREATION_ERROR =
       "Failed to construct a signer from supplied arguments.";
 
-  public CommandlineParser(final EthSignerBaseCommand baseCommand, final PrintStream output) {
+  public CommandlineParser(
+      final EthSignerBaseCommand baseCommand, final PrintWriter stdOut, final PrintWriter stdErr) {
     this.baseCommand = baseCommand;
-    this.output = output;
+    this.stdOut = stdOut;
```

Comment:
nit: not sure on naming, naming them literally stdOut and stdErr makes it hard for me to reason about what they are.
maybe they're errorOutput and ... standardOut? (yeah, that didn't help)

Reply:
we can discuss it in the code walkthrough.

Reply:
changes reverted